bentoml/server/bento_api_server.py

```diff
@@ -106,16 +116,18 @@ def start(self):
         # Flask dev server enabled threaded by default, disable it.
         self.app.run(port=self.port, threaded=False)
 
-    @staticmethod
-    def index_view_func():
+    def index_view_func(self):
```

Comment:
The best replacement for `@staticmethod` is `@classmethod`:

```
@classmethod
def index_view_func(cls):
    [...]
```

`cls` refers to the class, just like `self` refers to the instance.

Reply:
I was not sure how to use `@classmethod` so I just converted it back to `@staticmethod` and used it in same way as done for `docs_view_func`

Reply:
If `staticmethod` is an idiom in the codebase, then np.

But for the record, `classmethod` works as in my example above :). You just add the `@classmethod` decorator and then provide `cls` as the first argument (instead of `self` as for a normal member function).

Reply:
I tried that, but then is it possible to access `bentoservice` object without passing it?

Reply:
Aha, sorry, I hadn't seen that. Yeah, you did the right thing :)